plone/recipe/codeanalysis/__init__.py

```diff
@@ -198,6 +199,11 @@ def uninstall_pre_commit_hook(self):
 def code_analysis(options):
     start = time()
 
+    # if there is a second argument (first is always the progam itself)
```

Comment:
I don't mind adding it, I was just wondering which use cases will we use for it. I mean, do we want to add command line switches? like ``--no-flake8`` to not run flake8, or things like that?